### PR TITLE
build(github): Add getsentry acceptance tests

### PR DESCRIPTION
**Pull request by billyvg**
Follow-up to https://github.com/getsentry/sentry/pull/20167 to run `getsentry` acceptance tests.

Due to https://github.com/getsentry/getsentry/pull/4320 - we can run acceptance tests and hide traceback when py.test fails.

### CODE REVIEW

File: .github/workflows/getsentry.yml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+    - name: Collect dependencies
+      run: bin/test-utils/collect_deps
+
+    - name: Ensure image
+      run: bin/test-utils/ensure_image
+
+  acceptance:
+    name: acceptance
+    runs-on: ubuntu-latest
```
Comment (joshuarli) [inline .github/workflows/getsentry.yml:49]:
```suggestion
    runs-on: ubuntu-16.04
```

Just to be consistent. And avoid any surprises.

Comment (joshuarli) [reply .github/workflows/getsentry.yml]: Not sure if pinning the other ubuntu to the same is any faster or w/e.

File: .github/workflows/getsentry.yml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+      # The hostname used to communicate with the PostgreSQL from sentry
+      DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
+      # Number of matrix instances
+      TOTAL_TEST_GROUPS: 2
+      PGPASSWORD: postgres
+
+    services:
+      clickhouse:
+        image: yandex/clickhouse-server:19.17.4.11
```
[the remark this answers is not in the document]
Comment (joshuarli) [reply .github/workflows/getsentry.yml]: Would love to use devservices for this too, I might look into it.

File: .github/workflows/getsentry.yml [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+      kafka:
+        image: confluentinc/cp-kafka:5.1.2
+        env:
+          KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+          KAFKA_LISTENERS: INTERNAL://0.0.0.0:9093,EXTERNAL://0.0.0.0:9092
+          KAFKA_ADVERTISED_LISTENERS: INTERNAL://kafka:9093,EXTERNAL://kafka:9092
+          KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
+          KAFKA_INTER_BROKER_LISTENER_NAME: INTERNAL
+          KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
```
Comment (billyvg) [inline .github/workflows/getsentry.yml:133]: Hmm I may try replacing this with `sentry devservices`

File: .github/workflows/getsentry.yml (hunk 1 above)
Comment (joshuarli) [reply .github/workflows/getsentry.yml]: Oh man, there's 3 of them.

**Comment by billyvg** [comment]
Replaced by https://github.com/getsentry/sentry/pull/20167

**Pull request closed without merging**